README.md

```diff
@@ -253,6 +254,21 @@ value1: array.indexOf('array', 2), // 0
 value2: array.indexOf('array', 2, 2) // 3
 ```
 
+##### `array.invoke`
+wraps [`Ember.Array.invoke()`](http://emberjs.com/api/classes/Ember.Array.html#method_invoke), allows composing
+
+```js
+array: [{
+  foo(arg) {
+    return 'bar-' + arg;
+  }
+}],
+value1: array.invoke('array', 'foo', raw('baz')), // bar-baz
+value2: array.invoke('array', 'foo', 'arg'), // bar-hello
+value3: array.invoke('array', 'foo'), // bar-
```

Comment:
I think it would be helpful if the output examples shows that it returns an array of strings.

```js
value1: array.invoke('array', 'foo', raw('baz')), // ['bar-baz']
value2: array.invoke('array', 'foo', 'arg'), // ['bar-hello']
value3: array.invoke('array', 'foo'), // ['bar-']
```